spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+
+RSpec.describe Facility_Factory do
+    before(:each) do
+        @factory_1 = Facility_Factory.new
+        @colorado_locations = DmvDataService.new.co_dmv_office_locations
+        @new_york_locations = DmvDataService.new.ny_dmv_office_locations
+        @missouri_locations = DmvDataService.new.mo_dmv_office_locations
+        @co_dmvs = @factory_1.create_facilities(@colorado_locations)
+        @ny_dmvs = @factory_1.create_facilities(@new_york_locations)
+        @mo_dmvs = @factory_1.create_facilities(@missouri_locations)
+    end
+    
+    describe '#initialize' do
+        it 'can initialize' do 
+            expect(@factory_1).to be_an_instance_of (Facility_Factory)
+        end
+    end
+
+    describe '#it can create facilities' do
+        it 'creates instances of the facility class' do
+            expect(@co_dmvs).to all(be_an_instance_of (Facility))
+            expect(@co_dmvs[0]).not_to eq (@co_dmvs[4])
```

Comment:
Instead of checking these things are not equal, check to see that co_dmvs[0] is strictly equal to the instance you expect it to be.